hrpc/get.go

```diff
@@ -56,13 +56,16 @@ func (g *Get) Serialize() ([]byte, error) {
 			Row: g.key,
 		},
 	}
-	if g.family != nil {
-		get.Get.Column = []*pb.Column{
-			&pb.Column{
-				Family: g.family,
-				//Qualifier: [][]byte{[]byte("theCol")},
-			},
+	for family, qualifiers := range g.families {
+		bytequals := make([][]byte, len(qualifiers), len(qualifiers))
```

Comment:
This is equivalent to saying just `bytequals := make([][]byte, len(qualifiers))` – by default length==capacity for new slices.